packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -3452,6 +3472,132 @@ const TextStyle _messageStyle = TextStyle(
   height: 1.2,
 );
 
+class _ExitWidgetSelectionButtonWrapper extends StatefulWidget {
+  const _ExitWidgetSelectionButtonWrapper({
+    required this.button,
+    required this.buttonKey,
+  });
+
+  final Widget button;
+  final GlobalKey buttonKey;
+
+  @override
+  State<_ExitWidgetSelectionButtonWrapper> createState() =>
+      _ExitWidgetSelectionButtonState();
+}
+
+class _ExitWidgetSelectionButtonState
+    extends State<_ExitWidgetSelectionButtonWrapper> {
+  bool _toolTipVisible = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      alignment: AlignmentDirectional.topCenter,
+      children: <Widget>[
+        CustomPaint(
+          painter: _ExitWidgetSelectionTooltipPainter(
+            isVisible: _toolTipVisible,
+            buttonKey: widget.buttonKey,
+          ),
+        ),
+        MouseRegion(
+          onEnter: (_) {
+            setState(() {
```

Comment:
should this be on a short delay like the default tooltip behavior?

Reply:
Good point, done!